.github/workflows/pytest-gpu.yml

```diff
@@ -1,6 +1,5 @@
 # Runner information:
-# Standard NC24_Promo (24 vcpus, 224 GiB memory)
-# NVIDIA Tesla K80, Intel Xeon E5-2690 v3 (Haswell)
+# TODO: Summarize runner hardware for each config here
```

Comment:
Can you add something along the lines off:
```
* OpenACC on NVidia runs on `sarlaac`
* ...
```

I will create a page with the various hardware specifications in our wiki

Reply:
```yml
# Runner information:
# OpenACC on NVidia runs on `sarlaac`
# OpenACC on AMD runs on `nexu`
# OpenMP on NVidia runs on `bantha`
```
Ok?